### ISSUE TITLE
Extend API Endpoints for Hausumringe

### ISSUE THREAD

**Issue by op-hub** [amosproj/amos2024ss04-building-information-enhancer#151]
## Description
See Epic: #145 
## Depends on
#146
#147  

## Acceptance criteria
* FE request data from API
* Backend send data to FE

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release
